static/app/components/events/eventReplay/eventReplaySection.tsx

```diff
@@ -0,0 +1,12 @@
+import {EventDataSection} from 'sentry/components/events/eventDataSection';
+import {t} from 'sentry/locale';
+
+type EventReplaySectionProps = {children: JSX.Element};
+
+export function EventReplaySection({children}: EventReplaySectionProps) {
+  return (
+    <EventDataSection type="replay" title={t('Replay')}>
```

Comment:
We have some docs on this this, i think the one that applies is:
> Session Replay | This is the formal title of the product. Used when introducing the product as a concept. | Marketing title Product title (top of Index page) Docs title | always capitalized (never “session replay”)

```suggestion
    <EventDataSection type="replay" title={t('Session Replay')}>
```

https://www.notion.so/sentry/Session-Replay-Lexicon-Style-Guide-bcbbd569e5fc41f2aaff2f7d796fd401

Reply:
I think the title could be "Session Replay" because that's the product the section represents.

Inside the section there may or may-not be a 'replay event'.
- If there's a replay-event then we see the "Open Replay" button.
- If no replay-event, then we see the error message or onboarding or whatever.

That's what i'm thinking

Reply:
That works for me! Just wanted to make sure since the designs show "replay"